sdk/monitor/monitor-query/recordings/node/logsqueryclient_live_tests/recording_query_resource_centric_logs.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://api.loganalytics.io/v1/logs-arm-resource-id/query",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip,deflate",
+        "Authorization": "Sanitized",
+        "Connection": "keep-alive",
+        "Content-Length": "59",
+        "Content-Type": "application/json",
+        "User-Agent": "azsdk-js-monitor-query/1.0.4 azsdk-js-monitor-log-query/1.1.0 core-rest-pipeline/1.10.4 Node/v16.14.2 OS/(x64-Windows_NT-10.0.22621)",
+        "x-ms-client-request-id": "44dee56e-9b59-4d43-89c7-bc3fd51185c6"
+      },
+      "RequestBody": {
+        "query": "MyTable_CL | summarize count()",
+        "timespan": "P7D"
+      },
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Expose-Headers": "Retry-After,Age,WWW-Authenticate,x-resource-identities,x-ms-status-location",
+        "Connection": "keep-alive",
+        "Content-Length": "94",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Tue, 02 May 2023 00:04:09 GMT",
+        "Strict-Transport-Security": "max-age=15724800; includeSubDomains",
+        "Vary": "Accept-Encoding",
+        "Via": "1.1 draft-oms-848f8fcc7f-42tqh",
+        "X-Content-Type-Options": "nosniff"
+      },
+      "ResponseBody": {
+        "tables": [
+          {
+            "name": "PrimaryResult",
+            "columns": [
+              {
+                "name": "count_",
+                "type": "long"
+              }
+            ],
+            "rows": [
+              [
+                0
```

Comment:
We should try with a different query to get some rows back in the result.

Reply:
Yes, I need to fill in some data during the live testing for this. I'll attempt that in a separate PR